docs/csharp/language-reference/compiler-messages/warning-waves.md

```diff
@@ -5,6 +5,7 @@ ms.date: 05/11/2022
 f1_keywords:
   - "CS7023"
   - "CS8073"
+  - "CS8509"
```

Comment:
These two changes should be removed. The F1 help for this error shouldn't go to this page anymore.

```suggestion
```